docs/Maths.md

```diff
@@ -4,6 +4,10 @@
 ## Charts ##
 
 ### Using Matplotlib ###
+Before run this example, please install required dependencies following below command:
```

Comment:
```suggestion
Before running this example, please install the required dependencies using teh command below:
```

Reply:
Thanks for updating, sorry for my bad english

Reply:
No worry!
I'm not a native English speaker neither 😊